docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -22,13 +22,13 @@ Controlling the metrics helps you shape the checks and suites according to the s
 Default Metrics
 ===============
 All of the checks that evaluate model performance, such as
-:doc:`SingleDatasetPerformance </checks_gallery/vision/model_evaluation/plot_single_dataset_performance>`
+:doc:`TrainTestPerformance </checks_gallery/tabular/model_evaluation/plot_train_test_performance>`
 come with default metrics.
 
 The default metrics by task type are:
 
-Tabular
-_______
+Classification
+______________
```

Comment:
Now that there are several sections, maybe add a table in the beginning of all content (and to which subpackages it's relevant?)